commands/user_ls.go

```diff
@@ -1,22 +1,64 @@
 package commands
 
 import (
+	"encoding/json"
 	"fmt"
-
 	"github.com/MichaelMure/git-bug/cache"
 	"github.com/MichaelMure/git-bug/util/colors"
 	"github.com/MichaelMure/git-bug/util/interrupt"
 	"github.com/spf13/cobra"
 )
 
-func runUserLs(cmd *cobra.Command, args []string) error {
+var (
+	userLsOutputFormat string
+)
+
+func runUserLs(_ *cobra.Command, _ []string) error {
 	backend, err := cache.NewRepoCache(repo)
 	if err != nil {
 		return err
 	}
 	defer backend.Close()
 	interrupt.RegisterCleaner(backend.Close)
 
+	switch userLsOutputFormat {
+	case "org-mode":
+		return userLsOrgmodeFormatter(backend)
+	case "json":
+		return userLsJsonFormatter(backend)
+	case "plain":
+		return userLsPlainFormatter(backend)
+	case "default":
+		return userLsDefaultFormatter(backend)
+	default:
+		return fmt.Errorf("unknown format %s", userLsOutputFormat)
+	}
+}
+
+type JSONIdentity struct {
+	Id      string `json:"id"`
+	HumanId string `json:"human_id"`
+	Name    string `json:"name"`
+	Login   string `json:"login"`
+}
+
+func userLsPlainFormatter(backend *cache.RepoCache) error {
+	for _, id := range backend.AllIdentityIds() {
```

Comment:
Move the data loading in `runUserLs` instead of duplicating that in the formatters. It will also give you a better decoupling and testability.